docs/distributed/iam-import-with-openid.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+if [ -n "$TEST_DEBUG" ]; then
+	set -x
+fi
+
+pkill minio
+docker rm -f $(docker ps -aq)
+rm -rf /tmp/openid{1..4}
+
+export MC_HOST_myminio="http://minioadmin:minioadmin@localhost:22000"
+export MC_HOST_myminio1="http://dillon-svcacct-1:dillon-svcacct-1@localhost:22000"
```

Comment:
it does not seem we are creating a service account anywhere in this script. I will check if we can do that programmatically

Reply:
There is a service account pre created in the ZIP file getting imported. Yes it doesn't look too obvious. May be, will add a comment.